package.xml

```diff
@@ -9,16 +9,15 @@
 
   <buildtool_depend>cmake</buildtool_depend>
 
-  <depend>KimeraRPGO</depend>
-  <depend>GTSAM</depend>
-  <depend>gflags</depend>
-  <depend>glog</depend>
-  <depend>opencv</depend>
-  <depend>opengv</depend>
   <depend>dbow2</depend>
+  <depend>gtsam</depend>
+  <depend>kimera_rpgo</depend>
```

Comment:
Not ok, as I think it is KimeraRPGO, we should make a PR to the RPGO repo updating the name there.

Reply:
See https://github.com/MIT-SPARK/Kimera-RPGO/pull/45